src/annotator/integrations/html.js

```diff
@@ -16,6 +17,7 @@ import { HTMLMetadata } from './html-metadata';
  * by a more specific integration (eg. for PDFs).
  *
  * @implements {Integration}
+ * @implements {Destroyable}
```

Comment:
I wonder if `Integration` could imply/extend `Destroyable`? I'm not sure if this is supported when using JSDoc to define types.

Reply:
Indeed. I merged both types in this PR: https://github.com/hypothesis/client/pull/3454